.github/workflows/ci.yml

```diff
@@ -79,6 +79,12 @@ jobs:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
       - name: Run Tests
         run: pnpm test
+      - name: Upload test artifacts
+        if: ${{ !cancelled() }}
+        uses: actions/upload-artifact@v4
+        with:
+          name: playwright-report
+          path: '**/playwright-report/**/*'
```

Comment:
Do we maybe want to include [`**/test-results**/`](https://github.com/simonihmig/responsive-image/actions/runs/15495935234/job/43632602336#step:7:529) as well?

Reply:
> But these are also integrated into the report
Ah, nevermind then😅